src/bio/bio_internal.h

```diff
@@ -27,9 +27,9 @@
 #include <daos_srv/daos_server.h>
 #include <daos_srv/bio.h>
 
-#define BIO_DMA_PAGE_SHIFT	12		/* 4K */
+#define BIO_DMA_PAGE_SHIFT	12	/* 4K */
 #define BIO_DMA_PAGE_SZ		(1UL << BIO_DMA_PAGE_SHIFT)
-#define BIO_XS_CNT_MAX		48
+#define BIO_XS_CNT_MAX		48	/* Max VOS xstreams per blobstore */
```

Comment:
just to confirm that one blob store per NVME device, and "dss_tgt_nr == (number of VOS xtreams per blobstore) * (number of NVME device)"? if it is true seems the (number of VOS xtreams per blobstore) can be calculated when init?

Reply:
Each device can be assigned to different number of VOS xstreams. For instance, there are 3 VOS xstreams (dss_tgt_nr == 3) and 2 NVMe devices on a node, then one device will have 2 VOS xstreams and the other device will have 1 VOS xstream.